Documentation/ceph-cluster-crd.md

```diff
@@ -672,6 +672,8 @@ spec:
 
 ### External cluster
 
+**The minimum supported version of the External Cluster is Luminous 12.2.x.**
```

Comment:
Rook doesn't support Luminous for non-external clusters. Is it strange to support Luminous here?

Reply:
The thought is that if they have an existing Luminous cluster, Rook could connect to it and the CSI driver could connect to it for the rbd volumes. But Rook wouldn't be doing any configuration of ceph daemons if the external cluster is luminous (or some other unsupported version).